app/Controllers/userController.php

```diff
@@ -68,7 +68,7 @@ public function updateAction() {
 			));
 
 			if ($ok) {
-				$isSelfUpdate = Minz_Session::param('currentUser', '_') === $username;
+				$isSelfUpdate = Minz_Session::param(CURRENT_USER, '_') === $username;
```

Comment:
Some lines such as this one seems to have been forgotten for using `FreshRSS_Context::currentUser()`

Reply:
done